src/views/AddItem.jsx

```diff
@@ -12,24 +12,56 @@ const dayConverter = (text) => {
 	}
 };
 
-export function AddItem({ listId }) {
+export function AddItem({ listId, data }) {
 	const [itemName, setItemName] = useState('');
 	const [frequency, setFrequency] = useState('soon');
-	const [message, setMessage] = useState('');
+	const [itemMessage, setItemMessage] = useState('');
+	const [errorMessage, setErrorMessage] = useState('');
+
+	const clearMessage = (messageToAdjust) => {
+		setTimeout(() => {
+			messageToAdjust('');
+		}, 3000);
+	};
 
 	const handleSubmit = async (e) => {
 		e.preventDefault();
+
+		if (!itemName.trim()) {
+			setErrorMessage('Please enter item name.');
+			clearMessage(setErrorMessage);
+			return;
+		}
```

Comment:
`thought:` do we need to account for accidental characters? I'm only asking because I accidentally created an item by called `\`. It's really easy to do with a slip of your fingers on the keyboard. It made me wonder if we should use a regex to make sure that the item has at least one letter from the alphabet. It probably isn't in the scope of this ticket, as we want to "accept the user's input", but it was a thought that came to mind.

Reply:
Thanks for the attention to detail here! To me, the future issue involving `deleting items from my shopping list` would be used to "fix" this use case of accidentally adding a weird/not real item to the list. Letting the user fix their own mistake seems reasonable as long as we give them that functionality, and it avoids the slippery slope of attempting to let the app "over-correct" each user input by adding too much validation.

I'm curious what @ismarjiw and others think, though!

Reply:
Agreed 😊 I would love to learn regex for this use case but I wouldn't want us getting into those weeds 😅

Reply:
👏 love the discussion you're having here!

@tialaaa, good call on looking at what's coming next to decide whether to tackle @polly89's suggestion right now or not! Great work, love it!

Also, thanks for looping @ismarjiw too!